pukara-dev/culture
# PLU-14 creating a how to be a professional section based on the discussion we had

## Description

Please include a summary of the changes and the related issue(s). Also, mention any dependencies or context that might be helpful.

Fixes #(issue number)

## Type of change

- [✔] New feature

### Steps to Test (if applicable)

1. Run the project
2. Check the "about-us" section on the home page.

## Checklist

- [✔] I have performed a self-review of my code
- [✔] I have commented my code, particularly in hard-to-understand areas
- [✔] I have made corresponding changes to the documentation
- [✔] My changes do not introduce new warnings
- [✔] I have added tests that prove my fix is effective or that my feature works
- [✔] Any dependent changes have been merged and published in downstream modules
- [✔] Sensitive information (e.g., tokens, passwords) is not included.

## Additional context

This new section was added based on the discussion we had around the video "Aprenda a tocar PANDEIRO em 2 minutos".
